client/ayon_hiero/plugins/create/create_shot_clip.py

```diff
@@ -192,23 +224,36 @@ def get_instance_attr_defs(self):
             )
         ]
         if self.product_type == "plate":
-            instance_attributes.extend([
```

Comment:
Just to be sure, this means we do not want to let the user edit `vSyncOn` and `vSyncTrack` on the instance anymore ?
I think I still do not get how exactly those 2 features are connected. The way I understood it was:
* `reviewableSource` + `review` : control if a review h264 product gets created through OIIO and FFmpeg along the plate and from which source
* `vSyncOn` + `vSyncTrack`: define the hero track (main track) to construct a vertical synchronization.. not exactly sure what come after that

Reply:
Yes this is quite confusing I agree:

vSync attributes does not need to be user editable on instances. Those changes were not propagated anywhere and also this would be quite difficult to prapagate them in first place. It is much more easy if user will recreate those instances from scratch. 

Reviewable flaging is now doing following. Review attribute is saying if the plate needs to have acompanied reviwable representation and reviableSource is only saying what should be used for the representation creation.

Reply:
> Also very picky, but for UI/UX consistency, should we replicate the same UI between create attributes and instance attributes selection ? Currently we have 1 combo list with < none > entry in the creator then 1 checkbox + 1 combolist in the instance UI.

This is intentional. It would not make sense to have acctive review attribute with reviewableSource set to '< none >`.